### PR TITLE
Normalized stacked charts

### LINKED ISSUE #3246: Add proportional bars ("100%") option to bar charts
Allow to transform a chart like this:
image

Into one like this:
image

As you have the "Stacked" option today, I´d like to have a "proportional" option to allow for this.

Thanks!

### PR DESCRIPTION
**Pull request by tlrobinson**
Resolves #3246

### CODE REVIEW

**Comment by tlrobinson** [comment]
@salsakran @kdoh @mazameli  Here's every permutation of the option terminology I could come up with. Any preference?

screen shot 2016-11-02 at 12 11 42 pm
screen shot 2016-11-02 at 12 11 56 pm
screen shot 2016-11-02 at 12 11 32 pm
screen shot 2016-11-02 at 12 11 17 pm
screen shot 2016-11-02 at 12 11 01 pm
screen shot 2016-11-02 at 12 10 47 pm
screen shot 2016-11-02 at 12 10 28 pm
screen shot 2016-11-02 at 12 10 06 pm
screen shot 2016-11-02 at 12 09 46 pm

**Comment by salsakran** [comment]
I think I like "Stacked - 100%" best. 

For "None", maybe "No"? 

It also might read better if the heading was "Stacked?"

**Comment by mazameli** [comment]
This should be a set of radio options instead of a dropdown IMHO:

screen shot 2016-11-03 at 2 44 25 pm

**Comment by tlrobinson** [comment]
@mazameli  How's this? screen shot 2016-11-07 at 1 25 10 pm

I consolidated all of our Radio button components but left the horizontal style in (used in admin email settings and the dashboard "add card" modal)

@metabase/core-developers Anyone want to review the rest of this?

**Comment by mazameli** [comment]
LGTM 👌

**Comment by tlrobinson** [comment]
@metabase/core-developers Did anyone else want to review the code?

File: frontend/src/metabase/admin/settings/components/SettingsSetting.jsx [hunk 1]
```diff
@@ -31,15 +29,13 @@ export default class SettingsSetting extends Component {
...
-            );
-        });
-        return <ul className="flex text-grey-4">{options}</ul>
+        return (
+            <Radio
+                value={setting.value}
+                onChange={(value) => this.props.updateSetting(setting, value)}
+                options={setting.options}
+            />
+        );
```
Comment (salsakran) [inline frontend/src/metabase/admin/settings/components/SettingsSetting.jsx]: this is way cleaner.

**Review by salsakran — APPROVED** [review]
All looks pretty reasonable.

**Pull request merged**